Assets/AWSIM/Scripts/Vehicles/Vehicle.cs

```diff
@@ -245,7 +253,8 @@ void FixedUpdate()
             PreUpdateWheels();
 
             // Sleep ?
-            var sleep = CanSleep();
+            var maybe_sleep = CanSleep();
```

Comment:
I think it would be good to also consider conditions regarding time within the `CanSleep()` method. In other words, the `SleepTime()` method should not have to be called in `FixedUpdate()`

Reply:
I put the process of SleepTime() into CanSleep().